packages/oruga/src/components/sidebar/Inspector.vue

```diff
@@ -174,6 +174,20 @@ export default {
                         cmp.data.variant = 'warning';
                     }
                 },
+                {
+                    class: "visibleClass",
+                    description : 'Class of the sidebar when sidebar is visible',
+                    action: (cmp) => {
+                        this.position = 'static';
+                    }
+                },
+                {
+                    class: "hiddenClass",
+                    description : 'Class of the sidebar  when sidebar is hidden',
+                    action: (cmp) => {
+                        this.position = 'static';
```

Comment:
Why this?

Reply:
copy paste leftovers. I removed the action for the hidden class.